crates/re_renderer/shader/lines.wgsl

```diff
@@ -189,6 +190,20 @@ fn vs_main(@builtin(vertex_index) vertex_idx: u32) -> VertexOut {
         quad_dir = normalize(pos_data_quad_end.pos - pos_data_quad_begin.pos);
     }
 
+    // Resolve radius.
+    // (slight inaccuracy: End caps are going to adjust their center_position)
+    let camera_ray = camera_ray_to_world_pos(center_position);
+    let camera_distance = distance(camera_ray.origin, center_position);
+    var strip_radius = unresolved_size_to_world(strip_data.unresolved_radius, camera_distance, frame.auto_size_lines);
+    if draw_data.size_boost_in_points > 0.0 {
+        let size_boost = point_size_to_world(draw_data.size_boost_in_points, camera_distance);
+        strip_radius += size_boost;
+        // Push out positions as well along the quad dir.
+        // This is not only, but especially important if there's no miters on a line-strip (TODO(#829)),
```

Comment:
not only what?

Reply:
me grammar no good, thought this works :D. It is not **only** important when but **also** important when. 😄 
(fixing)